src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/TestUtils.java

```diff
@@ -394,6 +394,11 @@ public static String getResourceFilePath(String relPath) {
     }
 
     public static String getResponseBody(Response response) throws IOException {
```

Comment:
How do you like the idea that we change all methods in this class to `default` method in an interface? Then we could make `SQLIntegTestCase` implement it ("mixin"). Although mixin in Java is not as powerful as other language, I think we can start using it if you agree this is a good case.

Reply:
I think I never did that (default implementation in interface in java), so I am hesitant to use it now without properly understanding the intended usage. I will leave it as is at this time, but definitely will learn about that feature and how to properly use it.